Eatery/Controllers/Eateries/EateriesSharedViewController.swift

```diff
@@ -1,233 +0,0 @@
-//
-//  EateriesSharedViewController.swift
-//  Eatery
-//
-//  Created by Ethan Fine on 4/10/19.
-//  Copyright © 2019 CUAppDev. All rights reserved.
-//
-
-import AppDevAnnouncements
-import CoreLocation
-import NVActivityIndicatorView
-import UIKit
-
-class EateriesSharedViewController: UIViewController {
-
-    // Scroll
-
-    private var lastContentOffset: CGFloat = 0
-    private var showPillOnScrollStopTimer: Timer?
-
-    // View Controllers
-
-    private lazy var campusEateriesVC = CampusEateriesViewController()
-    private lazy var campusNavigationVC = EateryNavigationController(rootViewController: campusEateriesVC)
-    private lazy var collegetownEateriesVC = CollegetownEateriesViewController()
-    private lazy var collegetownNavigationVC = EateryNavigationController(rootViewController: collegetownEateriesVC)
-    private lazy var pillViewController = PillViewController(
-        leftViewController: campusNavigationVC,
-        rightViewController: collegetownNavigationVC
-    )
-
-    var activeNavigationController: EateryNavigationController {
-        if pillViewController.pillView.leftSegmentSelected {
-            return campusNavigationVC
-        } else {
-            return collegetownNavigationVC
-        }
-    }
-
-    var activeViewController: EateriesViewController {
-        pillViewController.pillView.leftSegmentSelected
-            ? campusEateriesVC
-            : collegetownEateriesVC
-    }
-
-    // Location
-
-    private lazy var locationManager: CLLocationManager = {
-        let locationManager = CLLocationManager()
-        locationManager.delegate = self
-        locationManager.desiredAccuracy = kCLLocationAccuracyBest
-        locationManager.startUpdatingLocation()
-        return locationManager
-    }()
-
-    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
-        super.init(nibName: nil, bundle: nil)
-    }
-
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
-    // MARK: View Controller
-
-    override func viewDidLoad() {
-        super.viewDidLoad()
-
-        setUpEateriesViewControllers()
-        setUpPillView()
-
-        pillViewController.delegate = self
-
-        // Present announcement if there are any new ones to present
-        presentAnnouncement { presented in
```

Comment:
Definitely worthwhile to continue to present announcements. Best place to move this would probably be in `EateryTabBarController`

Reply:
bump on making sure this gets added back!! (correct me if you actually did, and i just missed it)